src/js/events.js

```diff
@@ -24,12 +24,31 @@ function EventEmitter() {
 module.exports = EventEmitter;
 module.exports.EventEmitter = EventEmitter;
 
+EventEmitter.prototype._eventsCount = 0;
+EventEmitter.prototype._maxListeners = undefined;
+
+var defaultMaxListeners = 10;
```

Comment:
Should the default maximum number to be 10 in embeddable device for ShadowNode?

Reply:
I know less on embeddable device, but how much the default maximum number I should set, more or less? Or no need do this?
In my submission, the hardware of embeddable device is short board, if we set too more listeners, this will lead to memory failure. So we should set a smaller value. Like 8 or 5???

Reply:
Although I don't think there is not big difference between 7 and 10, it is also a good choice. @(｡･o･)@